### PR TITLE
Chore/axios

### PR DESCRIPTION
**Pull request by brayan15**
### Description
Configure axios instance and interceptors

### Steps
1) Run on browser console 
```javascript
localStorage.setItem("session", JSON.stringify({
    "refresh": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0b2tlbl90eXBlIjoicmVmcmVzaCIsImV4cCI6MTY5MTA4ODU2MywiaWF0IjoxNjg5NzkyNTYzLCJqdGkiOiI2MzE5OTJmYzE3YmU0M2E3OWQ1NmQ2OGU4NDBiZGNiMyIsInVzZXJfaWQiOiIxMzZjZTY5ZC01ZWJjLTRmMTctOGVhMi1mMTM1NDQ2ODI3OWYifQ.XODZ3gKr3FNwMW_GVt-JDrXKete_ZLFxe3ZLff02ek0",
    "access": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNjg5NzkzNDYzLCJpYXQiOjE2ODk3OTI1NjMsImp0aSI6IjRhNTdmZmMwOWExNjQ0YzViMGM0OWMyY2FlZTJmMWUyIiwidXNlcl9pZCI6IjEzNmNlNjlkLTVlYmMtNGYxNy04ZWEyLWYxMzU0NDY4Mjc5ZiJ9.YgBd9oR1SgCaQfwXIhJAQJbJg6gOXEjcrOHpIqUNos0"
}))
``` 
2) Run serve, open browse and inspect network
3) Go to `http://sebratech.recuirt.com:5173` and you can verify 3 calls of service 
```javascript
1) Call to skills endpoint is getting 401
2) Call to refresh endpoint and set the new token
3) Call to skills endpoint is getting 200
```

### CODE REVIEW

File: apps/nafasi/src/service/public.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const baseURL = "http://sebratech.recuirt.com:8000/api";
+
+const axiosPublic = axios.create({
+  baseURL,
+  headers: {
+    "Content-Type": "application/json",
+  },
+});
+
+const logut = () => localStorage.removeItem("session");
```
Comment (seballos) [inline apps/nafasi/src/service/public.js]: typo

File: apps/nafasi/src/service/public.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const login = async (params) => {
+  try {
+    const { data: session } = await axiosPublic.post("/token/", params);
+
+    localStorage.setItem("session", JSON.stringify(session));
+
+    return session;
+  } catch (_) {
```
Comment (seballos) [inline apps/nafasi/src/service/public.js]:
```suggestion
  } catch {
```

File: apps/nafasi/src/service/public.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    logut();
+
+    localStorage.setItem(
+      "session",
+      JSON.stringify({ refresh: session?.refresh || "", access: data.access })
+    );
+
+    return data.access;
+  } catch (_) {
```
Comment (seballos) [inline apps/nafasi/src/service/public.js]: same

File: apps/nafasi/src/hooks/useGetSkills.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  const [skills, setSkills] = useState([]);
+
+  const getSkills = async () => {
+    try {
+      setIsLoading(true);
+
+      const { data } = await restApiClient.getSkills();
+
+      setSkills(data);
+    } catch (_) {
```
Comment (seballos) [inline apps/nafasi/src/hooks/useGetSkills.js]: same

File: apps/nafasi/src/service/public.js (hunk 1 above)
Comment (brayan15) [reply apps/nafasi/src/service/public.js]: Done

File: apps/nafasi/src/service/public.js (hunk 2 above)
Comment (brayan15) [reply apps/nafasi/src/service/public.js]: Done

File: apps/nafasi/src/service/public.js (hunk 3 above)
Comment (brayan15) [reply apps/nafasi/src/service/public.js]: Done

File: apps/nafasi/src/hooks/useGetSkills.js (hunk 4 above)
Comment (brayan15) [reply apps/nafasi/src/hooks/useGetSkills.js]: Done

**Review by seballos — APPROVED** [review]
:ship:

**Pull request merged**